modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+Having a more advanced storage subsystem is essential to fully supporting critical database requirements. The Container Storage Interface (CSI) allows {product-title} to expand storage volumes after they have already been created.
```

Comment:
Is this first sentence necessary? If it is, more advanced than what?

Reply:
No, not necessary. I removed the first sentence.